openshift/jenkins-persistent-template.json

```diff
@@ -80,7 +99,7 @@
             "containers": [
               {
                 "name": "jenkins",
-                "image": "${JENKINS_IMAGE}:${JENKINS_IMAGE_VERSION}",
+                "image": " ",
```

Comment:
This file is taken from OpenShift catalogue. This is the template that is used when you select "Jenkins Persistent" when creating a new project in OpenShift.

The image is empty because of the image stream above.